frappe/utils/backups.py

```diff
@@ -419,15 +421,15 @@ def take_dump(self):
 		extra = []
 		if self.db_type == "mariadb":
 			if self.backup_includes:
-				extra.extend([f"'{x}'" for x in self.backup_includes])
+				extra.extend(self.backup_includes)
 			elif self.backup_excludes:
-				extra.extend([f"--ignore-table='{self.db_name}.{table}'" for table in self.backup_excludes])
```

Comment:
`table` might contain spaces, so the quotes here were justified, right?

Reply:
Yes, they should still be escaped properly by `cmd.append(shlex.join(args))`.

If that wasn't the case, it would have been, in my understanding, caught by https://github.com/frappe/frappe/blob/70867b47a41b94337e0e436a5e844417d1ffefa6/frappe/tests/test_commands.py#L792-L796

using:
```python
	backup_map = {
		"includes": {
			"includes": [
				"ToDo",
				"Note",
			]
		},
		"excludes": {"excludes": ["Activity Log", "Access Log", "Error Log"]},
	}

```